README.md

```diff
@@ -449,6 +449,68 @@ Note that because this method is not available in the generic `IDataReader` inte
 TimeSpan timeSpanTime = ((SnowflakeDbDataReader)reader).GetTimeSpan(13);
 ```
 
+Executing a Batch of SQL Statements (Multi-Statement Support)
+--------------------------------------------------------------
+
+With the .NET 2.0.18 and later connector, you can send
```

Comment:
"later connector" sounds odd. Maybe "With version 2.0.18 and later of the .NET connector".

Reply:
Done